web-test-runner.config.js

```diff
@@ -124,7 +124,7 @@ export default {
 
   // If a test suite takes longer than this, it's almost certainly hanging and
   // won't finish. 2 minutes is the default.
-  testsFinishTimeout: process.env.CI ? 120_000 : 60_000,
+  testsFinishTimeout: process.env.CI ? 180_000 : 60_000,
```

Comment:
Got [snagged](https://github.com/CrowdStrike/glide-core/actions/runs/16526868405) on a timeout. So bumped the timeout up a bit.

I'm [planning](https://github.com/CrowdStrike/glide-core/pull/986#discussion_r2223344062) to break apart out tests suites in the near future. So we should be able to reduce this number soon by quite a bit.